source/ways-of-working/new-component/github-repo.html.md.erb

```diff
@@ -40,3 +40,14 @@ Any developer added to [hmcts org](https://github.com/hmcts) is able to create a
    - Enable `Require a pull request before merging` and `Require approvals`
    - Choose which status checks needs to be mandatory for merging PRs (this can be done only after checks like Jenkins are setup)
    - We recommend enabling `Include administrators`.
+
+## Suggested Branch Protection Rules
```

Comment:
this duplicates the section above, please look at merging / removing above

Reply:
Dam this could have saved time.
I have removed my new section and added some extra parts for the existing.